DWIConvert/DWIConverter.cxx

```diff
@@ -66,7 +66,12 @@ void DWIConverter::ConvertToMutipleBValuesUnitScaledBVectors()
     for( unsigned int k = 0; k < m_DiffusionVectors.size(); ++k )
     {
       double mag = m_DiffusionVectors[k].magnitude();
+
+      //0.01 tolerance will lead restore bvalue has about 30 error.
```

Comment:
Hui,

I don't know what "30 error" is. 30 milimters of error?  Describe the problem so that it makes sense in 2 years from now., and ONLY describe the current code, not the old code.

These comments are very confusing.  In 2 years from now, it will be completely confusing about what these comments mean.  In 1 week from now, we may add a new test 1, and the current test 42 will become the new test 43.

Please remove these comments or fix them.